back-end/resources/room_config.json

```diff
@@ -0,0 +1,251 @@
+{
+  "general": {
+    "name": "Escape X",
+    "duration": "00:30:00",
+    "host": "192.0.0.84",
+    "port": 1883
+  },
+  "devices": [
+    {
+      "id": "controlBoard",
+      "description": "Control board with three switches, three slides with lights and one main switch.",
+      "input": {
+        "redSwitch": {
+          "type": "boolean"
```

Comment:
type is unnecessary, key/id: type is  sufficient, example: "redSwitch": "boolean"
value is unnecessary, this is the same as defining one component and calling it value